Assignment3/MyDictionary.java

```diff
@@ -0,0 +1,30 @@
+package Assignment3;
+
+import java.util.ArrayList;
+
+public class MyDictionary {
+	ArrayList<String> words;
+	ArrayList<String> prefixes;

```

Comment:
FYI - this isn't really part of the assignment, but because of repeated contains checks this would be much better as a HashSet ( O(1) contains time) than an ArrayList ( O(n) contains time )